setup.py

```diff
@@ -458,10 +493,17 @@ def build_libraries(self, libraries):
                 cflags = [f for f in cflags if not f.endswith('openmp')]
 
             prefix = '/' if self.compiler.compiler_type == 'msvc' else '-'
-            build_info['cflags'] = [
+            cflags = [
                 f for f in cflags if f.startswith(prefix)
             ]
 
+            # Patch C++20 flag for older gcc/clang support
+            if '-std=c++20' in cflags:
+                if config.cpp20_compile_arg is None:
+                    raise RuntimeError("Cannot compile C++20 code with current compiler")
+                cflags = [f for f in cflags if f != '-std=c++20'] + [config.cpp20_compile_arg]
```

Comment:
Allows to change `-std=c++20` to `-std=c++2a` for gcc and clang < 10

Reply:
I don't quite understand this change, but I do want to point out that SPERR compiles with -std=c++17 too, on systems without C++20 support. (There's a small performance difference.)